src/styles/_catalog.scss

```
@@ -14,6 +13,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms;
```

Bit risky in general, but okay here I suppose

---

If you transition just two properties - it's better to set them individually.
```suggestion
    transition: color 300ms, transform 300ms;
```